project3/index.html

```diff
@@ -0,0 +1,410 @@
+                    <p>Please select as many as apply to your origin and how you gained your powers.</p>
+
+                    <span class=" category">
+                        <input name="categroy" id="alien" type="checkbox">
+                        <label for="alen">Alien</label></span>
+
+                    <span class="category">
+                        <input name="category" id="cyborg" type="checkbox">
+                        <label for="cyborg">Cyborg</label></span>
+
+                    <span class="category">
+                        <input name="category" id="experiment" type="checkbox">
+                        <label for="experiment">Experiment</label></span>
+
+                    <span class="category">
+                        <input name="category" id="human" type="checkbox">
+                        <label for="human">Human</label></span>
+
+                    <span class="category">
+                        <input name="category" id="inhuman" type="checkbox">
+                        <label for="inhuman">Inhuman</label></span>
+
+                    <span class="category">
+                        <input name="category" id="mutant" type="checkbox">
+                        <label for="mutant">Mutant</label></span>
+                </div>
+            </fieldset>
+
+            <fieldset>
+                <div class="gender">
+                    <legend>Gender</legend>
+
+                    <span class="category">
+                        <input name="gender" id="female" type="radio" required>
+                        <label for="female">Female</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="male" type="radio" required>
+                        <label for="male">Male</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="nonbinary" type="radio" required>
+                        <label for="nonbinary">Nonbinary</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="genderqueer" type="radio" required>
+                        <label for="genderqueer">Genderqueer/Genderfluid</label></span>
+
+                    <span class="category">
+                        <input name="gender" id="answer" type="radio" required>
+                        <label for="answer">Prefer not to answer/label></span>
+
+                    <span class="category">
+                        <input name="gender" id="describe" type="radio" required>
+                        <label for="describe">Prefer to self describe</label></span>
+                    <textarea name="describe" id="describe" cols="30" rows="10"></textarea>
+                </div>
+            </fieldset>
+
+            <h3 class="pna">powers & abilities</h3>
+
+            <label for="powers">Describe your powers</label>
+            <textarea name="powers" id="powers" cols="30" rows="10"></textarea>
```

Comment:
Same note as above re: `cols` and `rows`
```suggestion
            <textarea name="powers" id="powers"></textarea>
```